xdsl/dialects/stencil.py

```diff
@@ -1345,8 +1345,7 @@ def verify_(self) -> None:
             return
         if not isinstance(self.temp.owner, ApplyOp | CombineOp):
             raise VerifyException(
-                f"Expected stencil.buffer to buffer a stencil.apply or stencil.combine's output, got "
-                f"{self.temp.owner}"
+                "Expected stencil.buffer operand to be a result of stencil.apply or stencil.combine"
```

Comment:
I think it would be reasonable to include the operations name here, which shouldn't need `repr` stuff.